.gitignore

```diff
@@ -7,6 +7,7 @@ dist/
 
 # misc
 .cache
+.parcel-cache
```

Comment:
The parcel cache directory has changed from `.cache` to `.parcel-cache`.

https://parceljs.org/getting-started/migration/#cache-location

I've kept the old one around in `yarn build-clean` and in `.gitignore` so that users who have already cloned this repo won't run into issues.